DigitalLearningSolutions.Data.Migrations/202201171130_AddUniqueConstraintToCompetencyAssessmentQuestionRoleRequirements.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202201171130)]
+    public class AddUniqueConstraintToCompetencyAssessmentQuestionRoleRequirements : Migration
+    {
+        public override void Up()
+        {
+            Create.UniqueConstraint(

```

Comment:
Is it worth adding a DELETE query in here to remove any existing records that won't fit the new constraint? i.e. any CompetencyAssessmentQuestionRoleRequirements that aren't the first record with the same SelfAssessmentID,                CompetencyID, AssessmentQuestionID, LevelValue combination - just in case?

Reply:
Discussed this with Steve. Ignore this too!